src/qt/intro.h

```diff
@@ -72,7 +76,7 @@ private Q_SLOTS:
     //! Total required space (in GB) depending on user choice (prune or not prune).
     int64_t m_required_space_gb{0};
     uint64_t m_bytes_available{0};
-    const int64_t m_prune_target_gb;
+    int64_t m_prune_target_gb;
```

Comment:
maybe still worth having `{0}` in case some wild new constructor overload appears?